### PR TITLE
Call view accessibility fixes

### PR DESCRIPTION
**Pull request by t3chguy**
Fixes https://github.com/vector-im/element-web/issues/18516

---
Here's what your changelog entry will look like:

## 🐛 Bug Fixes
 * Call view accessibility fixes ([\#7439](https://github.com/matrix-org/matrix-react-sdk/pull/7439)). Fixes vector-im/element-web#18516.

Preview: https://61c46440dcb3899e29c98f2e--matrix-react-sdk.netlify.app
⚠️ Do you trust the author of this PR? Maybe this build will steal your keys or give you malware. Exercise caution. Use test accounts.

### CODE REVIEW

**Review by SimonBrandner — APPROVED** [review]
Thanks!

File: res/css/views/voip/CallView/_CallViewButtons.scss [hunk 1]
```diff
@@ -16,6 +16,13 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+// data-whatintent makes more sense here semantically but then the tooltip would stay visible without the button
+// which looks broken, so we match the behaviour of tooltips which is fine too.
+[data-whatinput="mouse"] .mx_CallViewButtons.mx_CallViewButtons_hidden {
+    opacity: 0.001; // opacity 0 can cause a re-layout
```
Comment (turt2live) [inline res/css/views/voip/CallView/_CallViewButtons.scss:22]: does this cause issues in Firefox? It's been known to disagree with floats on some properties, particularly at low numbers like this.

**Review by turt2live — APPROVED** [review]
overall seems fine - thanks!

File: res/css/views/voip/CallView/_CallViewButtons.scss (hunk 1 above)
Comment (t3chguy) [reply res/css/views/voip/CallView/_CallViewButtons.scss:22]: No idea, I'm maintaining the current impl, just tweaking when its shown/hidden

**Pull request merged**